R/validations.R

```diff
@@ -194,12 +198,13 @@ validate_has_elements <- function(x, msg) {
 
 #' Validates no intersection between two vectors
 #'
-#' @description `r lifecycle::badge("stable")`
-#' @param x `vector`
-#' @param y `vector`
-#' @param msg message to display if `x` and `y` intersect
+#' `r lifecycle::badge("stable")`
```

Comment:
Won't this push the second paragraph to `@details` section?

```suggestion
#' @description `r lifecycle::badge("stable")`
```